library/ssl_tls13_keys.c

```diff
@@ -1537,7 +1537,8 @@ int mbedtls_ssl_tls13_compute_resumption_master_secret( mbedtls_ssl_context *ssl
 
     MBEDTLS_SSL_DEBUG_BUF( 4, "Resumption master secret",
              ssl->session_negotiate->app_secrets.resumption_master_secret,
-             mbedtls_md_get_size( mbedtls_md_info_from_type( md_type ) ) );
+             PSA_HASH_LENGTH(
+                PSA_ALG_HMAC( mbedtls_hash_info_psa_from_md( md_type ) ) ) );
```

Comment:
```suggestion
                mbedtls_hash_info_psa_from_md( md_type ) ) );
```
not sure why we would need PSA_ALG_HMAC here. Then to do the MD type to PSA alg translation, just above we use `mbedtls_psa_translate_md( md_type )` thus I would suggest to use the same here, thus:
```suggestion
                mbedtls_psa_translate_md( md_type ) ) );
```